syncer_test.go

```diff
@@ -29,6 +29,13 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// Cleanup helper to remove tmp files we've (maybe) created
+func cleanup(syncer *Syncer) {
+	for _, entry := range syncer.clients {
+		os.RemoveAll(entry.WriteDirectory)
```

Comment:
If we always make a new temporary directory to run tests in, then we won't need to worry about that.  Seems like a safer solution than adding warnings